src/app/nextversion/nextversion.go

```diff
@@ -100,7 +115,19 @@ func NextVersion(cCtx *cli.Context) error {
 		return err
 	}
 
+	entryCap, err := bump.NameToType(cCtx.String(BumpCapFlag))
+	if err != nil {
+		return fmt.Errorf("error with the changes' cap: %w", err)
+	}
+	dependencyCap, err := bump.NameToType(cCtx.String(DependencyCapFlag))
+	if err != nil {
+		return fmt.Errorf("error with dependencies' cap: %w", err)
```

Comment:
```suggestion
		return fmt.Errorf("parsing dependency bump cap: %w", err)
```

Reply:
I like more your wording. solved in [#0eb5b56](https://github.com/newrelic/release-toolkit/pull/128/commits/0eb5b567f41c0b40ab9bc50adde5facdb51cae3b)